### PR TITLE
Stop using Boolean constructors

### PR DESCRIPTION
**Pull request by Emkas**
In `ValueHolderTestCaseBase`.

### CODE REVIEW

**Review by melloware — APPROVED** [review]

**Comment by pizzi80** [comment]
please approve all these cleaning/improving PRs

I'm doing a similar work on other classes 
and I would like to merge at some point or we will go in conflict

**Comment by Emkas** [comment]
@pizzi80 wow, good to know! In one of PR I wrote that I corrected nearly half of warnings in this project, but I'm also waiting for approval in several of them so I can push another.

But since you've wrote, I'll try to push it without waiting.  It we'll be out waste of time if we would do the same job 😀

On Monday I'll look into your changes and I will stop doing mine for now.

**Comment by pizzi80** [comment]
Now I'm mainly working on faces.js ... in meantime I'll resist from change Java files :smiley: 

so with some luck We should not be in conflict :dancers:

**Pull request merged**
